config/config-example.js

```diff
@@ -379,3 +379,8 @@ exports.grouplist = [
 		ip: 's',
 	},
 ];
+
+// noCustomGroupList - set this to false if your server has custom ranks
+// it will send an additional |authconfig| message on connection for the
+// client to pick up the additional non-conventional ranks.
+exports.noCustomGroupList = true;
```

Comment:
Specifically, this is a micro-optimization that no one is going to need, and is going to cause way more annoyance than necessary. This should also default to `false`.